projects/extension/src/containers/Options.tsx

```diff
@@ -259,82 +129,48 @@ const Options: React.FunctionComponent = () => {
-        TabIndicatorProps={{
-          style: {
-            display: "none",
-          },
-        }}
-      >
-        <MenuTab label="Networks"></MenuTab>
-        <MenuTab label="Settings"></MenuTab>
-        <MenuTab label="Logs"></MenuTab>
-      </MenuTabs>
-      <TabPanel value={value} index={0}>
-        {networks.length ? (
-          networks.map((network: NetworkTabProps, i: number) => {
-            const { name, health, apps } = network
-            return (
-              <NetworkTab key={i} name={name} health={health} apps={apps} />
-            )
-          })
-        ) : (
-          <div>No networks or apps are connected to the extension.</div>
-        )}
-      </TabPanel>
-      <TabPanel value={value} index={1}>
-        <FormControl component="fieldset">
-          <FormGroup aria-label="position" row>
-            <FormControlLabel
-              value="Notifications:"
-              control={
-                <Switch
-                  checked={notifications}
-                  onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
-                    setNotifications(e.target.checked)
-                  }
-                  color="primary"
-                  name="checkedB"
-                  inputProps={{ "aria-label": "primary checkbox" }}
-                />
-              }
-              label="Notifications:"
-              labelPlacement="start"
-            />
-          </FormGroup>
-        </FormControl>
-      </TabPanel>
-      <TabPanel value={value} index={2}>
-        <div className={classes.logs}>
-          <div className={classes.logTitle}>
-            <Button
-              variant="contained"
-              startIcon={poolingLogs ? <PauseIcon /> : <PlayArrow />}
+    <div className="font-roboto my-8 mx-12">
+      <div className="pb-10 text-base">
+        <Logo textSize="lg" />
+      </div>
+      <Tabs tabTitles={["Networks", "Logs"]}>
+        <section>
+          {networks.length ? (
+            networks.map((network: NetworkTabProps, i: number) => {
+              const { name, health, apps } = network
+              return (
+                <NetworkTab key={i} name={name} health={health} apps={apps} />
+              )
+            })
+          ) : (
+            <div>No networks or apps are connected to the extension.</div>
+          )}
+        </section>
+        <section className="block">
+          <div className="my-5 flex">
+            <button
+              className="border rounded-md px-2 bg-stone-200	hover:bg-stone-400"
               onClick={() => setPoolingLogs(!poolingLogs)}
             >
               {poolingLogs ? "Pause" : "Retrieve "} logs
-            </Button>
-            <Button
-              className={classes.copyClipboard}
-              variant="contained"
-              startIcon={<FileCopyIcon />}
+            </button>
+            <button
+              className="my-0 mx-2 border rounded-md px-2 bg-stone-200	hover:bg-stone-400"
```

Comment:
erroneous tab in the class list.  Also you're using the same class list above on line 152 without `my-0`.  Maybe putting the only extra class (`my-0`) at the end of the class list here will make that more obvious when you're scanning through the file.

Reply:
Its a very good idea to actually `autosort` all the classNames in order to be more efficient.